lib/events/test/suite.go

```
@@ -142,7 +146,13 @@ func (s *EventsSuite) SessionEventsCRUD(c *check.C) {
 		time.Sleep(s.QueryDelay)
 	}
 
-	history, _, err := s.Log.SearchEvents(s.Clock.Now().Add(-1*time.Hour), s.Clock.Now().Add(time.Hour), defaults.Namespace, nil, 100, "")
+	var history []events.AuditEvent
+
+	utils.RetryStaticFor(time.Minute*5, time.Second*5, func() error {
```

`RetryStaticFor` error is not checked.

---

Fixed